templates/default/index.html.twig

```diff
@@ -7,8 +7,6 @@
         <p class="intro">{{ 'page.index.body.intro'|trans }}</p>
         <br>
 
-        <p>{{ 'page.index.body.sdk_discription'|trans({'%idp-entity-id%': url('gssp_saml_metadata', {})}) }}</p>
-
```

Comment:
No need to remove this section in my opinion

Reply:
Do you perhaps know why it was removed from MFA?

Reply:
While setting up the project, I started removing everything I thought we would not be needing. And that was one of the sdk description.

Reply:
Thanks for the clarification, will restore it